src/main/java/com/coherentsolutions/training/aqa/java/web/nikonova/objects/BasePage.java

```diff
@@ -0,0 +1,17 @@
+package com.coherentsolutions.training.aqa.java.web.nikonova.objects;
+
+import com.coherentsolutions.training.aqa.java.web.nikonova.browsers.BrowsersSettings;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class BasePage {
+
+    protected WebDriver driver;
+
+    public BasePage(WebDriver driver) {
+        this.driver = driver;
+
+        PageFactory.initElements(BrowsersSettings.getDriver(), this);
```

Comment:
No need to get the same driver from BrowserSettings class, you can use existing instance of the driver PageFactory.initElements(driver, this);